desktop/get-started.md

```diff
@@ -3,33 +3,28 @@ description: Docker Dashboard
 keywords: Docker Dashboard, manage, containers, gui, dashboard, images, user manual
 title: Sign in and get started
 redirect_from:
-- /docker-for-windows/
-- /docker-for-windows/index/
-- /docker-for-windows/started/
```

Comment:
Slightly more "code churn" in this page, but I sorted them alphabetically (which should help with future updates to see if there's no duplicates)